scripts/bash/downloadEnvironmentVariables.sh

```diff
@@ -19,9 +19,7 @@ for PACKAGE in $PACKAGES; do
     ENV_FILE_PATH=${DIR}/../../packages/${PACKAGE}/.env
 
     # checkout branch specific env vars, or dev as fallback, temporarily redirecting stderr
-    exec 3> /dev/stderr 2> /dev/null
-    lpass show --notes ${PACKAGE}.${BRANCH}.env > ${ENV_FILE_PATH} || lpass show --notes ${PACKAGE}.dev.env > ${ENV_FILE_PATH}
-    exec 2>&3
+    lpass show --notes ${PACKAGE}.${BRANCH}.env 2> /dev/null > ${ENV_FILE_PATH} || lpass show --notes ${PACKAGE}.dev.env > ${ENV_FILE_PATH}
```

Comment:
Stderr now redirected specifically for the first "branch specific" one, which often fails before we fall back to dev